src/assets/checkmark.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Please consider cleaning up these SVG elements, since there's a lot of noise coming from things you don't actually need.